src/components/PhonicIcon/PhonicIcon.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../pages/Index/index.scss";
+.PhonicIcon {
+    width: 20vw;
+    height: 20vw;
+    margin: 2em;
+    padding: 20px;
+    background-color: white;
+    border-radius: 2em;
+    box-shadow: 0em .2em #e6e6e6;
+    display: box;
+  }
+.PurplePhonicIcon {
+  width: 20vw;
+  height: 20vw;
+  margin: 2em;
+  padding: 20px;
```

Comment:
i feel like maybe you could have 1 class with all the stuff thats in both and then have the color be their own classes? works either way but not a huge fan of the repetition